pkgs/development/python-modules/deal/default.nix

```diff
@@ -77,6 +77,8 @@ buildPythonPackage rec {
     "test_scheme_contract_is_satisfied_when_setting_arg"
     "test_scheme_contract_is_satisfied_within_chain"
     "test_scheme_errors_rewrite_message"
+    # broken since pytest > 7.1.3
```

Comment:
Hi Sandro,
the error message is:
```
_____________________________ test_exception_hook ______________________________

capsys = <_pytest.capture.CaptureFixture object at 0x7ffff1d97bb0>

    def test_exception_hook(capsys):
        pre_path = str(Path('deal', '_runtime', '_contracts.py'))
        f = deal.pre(lambda x: x > 0)(lambda x: x)
        with pytest.raises(deal.PreContractError) as exc_info:
            f(-2)

        # custom hook is registered
        assert sys.excepthook is exception_hook

        # the default hook shows the full traceback
        _excepthook(exc_info.type, exc_info.value, exc_info.tb)
        captured = capsys.readouterr()
        assert captured.out == ''
        assert pre_path in captured.err

        # the custom hook hides deal from the traceback
        exception_hook(exc_info.type, exc_info.value, exc_info.tb)
        captured = capsys.readouterr()
        assert captured.out == ''
        # the hook doesn't work on python 3.6 and below
        if sys.version_info[:2] >= (3, 7):
>           assert pre_path not in captured.err
E           assert 'deal/_runti...contracts.py' not in 'Traceback (...-2\x1b[0m)\n'
E             'deal/_runtime/_contracts.py' is contained here:
E               ld/source/deal/_runtime/_contracts.py", line 87, in wrapper
E                   return contracts._run_sync(args, kwargs)
E                 File "/build/source/deal/_runtime/_contracts.py", line 119, in _run_sync
E                   validator.validate(args, kwargs)
E                 File "/build/source/deal/_runtime/_validators.py", line 175, in _init
E                   self.validate(args, kwargs, exc)...
E
E             ...Full output truncated (4 lines hidden), use '-vv' to show

tests/test_exceptions.py:201: AssertionError
```
So now the custom hook doesn't seem to hide `deal` from the stacktrace. My guess was that pytest now includes more output in the stacktrace. I haven't found anything conclusive on the changelog for pytest though. Also all other tests pass and I'm not sure how relevant this error is. 
Also since there doesn't seem to be a way to post issues to upstream, this could potentially take some time to fix the test upstream.
Therefore I excluded the test